packages/web/src/utils/utils.ts

```diff
@@ -108,3 +108,23 @@ export const emitEvent = (
 
   return true;
 };
+
+// Log validation error for required properties in components
+export const logError = (
+  name,
+  errorArr,
+  optionalAttrsArrToRemove
+) => {
```

Comment:
Can we add typescript definitions for the parameters?
```suggestion
export const logError = (
  name: string,
  errorArr: [],
  optionalAttrsArrToRemove?: []
) => {
```